lms/static/scripts/frontend_apps/components/FilePickerApp.tsx

```diff
@@ -495,6 +516,26 @@ export default function FilePickerApp({ onSubmit }: FilePickerAppProps) {
                             )
                           }
                         />
+                        <Popover
+                          open={maxPointsPopoverOpen}
+                          anchorElementRef={iconRef}
+                          onClose={() => setMaxPointsPopoverOpen(false)}
+                          classes="p-2"
+                          placement="above"
+                          arrow
+                        >
+                          <div className="flex flex-col gap-y-2">
+                            (Optional) Add a Max Points value here instead of
+                            using your LMS grading settings.
+                            <Link
+                              href="https://web.hypothes.is/help/max-points-in-hypothesis-enabled-readings/"
+                              underline="always"
+                              target="_blank"
+                            >
+                              Learn more about grading options
+                            </Link>
+                          </div>
+                        </Popover>
```

Comment:
It would be nice to extract the info button into a separate component in future, both to avoid `FilePickerApp` growing too complex and also for re-use. I did a quick experiment but found some layout issues and style issues that would need to be worked through.